app/rss.py

```diff
@@ -0,0 +1,62 @@
+import xml.etree.ElementTree as ET
+
+from pathlib import Path
+from podgen import Podcast, Episode, Person, Media
+
+from flask import Response
+from app import app
+
+from . import models
+
+file = Path('app/templates/feed_template.xml')
```

Comment:
Is it a constant? If so, it needs to be uppercased

Reply:
Also, `file` is a builtin, one can get into all kinds of troubles after redefining builtins.